### ISSUE TITLE
KeePassXC doesn't start at all since 2.7.9

### ISSUE THREAD

**Issue by QuestionMarkT** [keepassxreboot/keepassxc#10971]
I have 3 different machines which one of them is a virtual machine, KPXC 2.7.7 and 2.7.8 worked fine on all 3 machines, 2.7.9 works fine on just only one. The remaning two can't run KeePassXC and I was able to find this log in the Event Viewer on one of the machines:
### Application popup: KeePassXC.exe - Application Error : The instruction at 0x00007FF9F11C2F58 referenced memory at 0x0000000000000000. The memory could not be read.
Operating System: noticed this issue on Windows Server 2016 (14393.7070) and Windows 11 23H2 (22631.3737) but on some other Windows 11 23H2 (22631.3447) surprisingly works fine

edit: I was able to find logs on 2nd machine that should be more helpful:

Faulting application name: KeePassXC.exe, version: 2.7.9.0, time stamp: 0x6673531d
Faulting module name: MSVCP140.dll, version: 14.32.31332.0, time stamp: 0x8b64702c
Exception code: 0xc0000005
Fault offset: 0x00000000000132f8
Faulting process id: 0x0x4FB8
Faulting application start time: 0x0x1DAC675C9C6E06F
Faulting application path: C:\Users\\\<REDACTED>\Desktop\KeePassXC-2.7.9-Win64\KeePassXC.exe
Faulting module path: C:\WINDOWS\SYSTEM32\MSVCP140.dll

**Comment by rickjdev** [comment]
Same here. Tried uninstalling and reinstalling, no luck. Uninstalled and went back to 2.7.8, no problem.

Here's the error I see on Windows 11:

Faulting application name: KeePassXC.exe, version: 2.7.9.0, time stamp: 0x6673531d
Faulting module name: MSVCP140.dll, version: 14.32.31332.0, time stamp: 0x8b64702c
Exception code: 0xc0000005
Fault offset: 0x00000000000132f8
Faulting process id: 0x0x2408
Faulting application start time: 0x0x1DAC6761B8F6822
Faulting application path: C:\Program Files\KeePassXC\KeePassXC.exe
Faulting module path: C:\WINDOWS\SYSTEM32\MSVCP140.dll
Report Id: e71c1ff2-cf53-4ab6-9a19-644a5c60ee90
Faulting package full name: 
Faulting package-relative application ID:

**Comment by droidmonkey** [comment]
Github search must be broken today....

https://github.com/keepassxreboot/keepassxc/issues/10925#issuecomment-2179741873

**Comment by thund3rb1rd78** [comment]
> Github search must be broken today....
> 
> [#10925 (comment)](https://github.com/keepassxreboot/keepassxc/issues/10925#issuecomment-2179741873)

surely you're joking

**Comment by jmssil** [comment]
I just got it now, but I have not even updated KPXC today.

Faulting application name: KeePassXC.exe, version: 2.7.9.0, time stamp: 0x6673531d
Faulting module name: MSVCP140.dll, version: 14.29.30133.0, time stamp: 0x60ff28cc
Exception code: 0xc0000005
Fault offset: 0x0000000000013020
Faulting process id: 0x0x2B84
Faulting application start time: 0x0x1DB09017A2A2ADB
Faulting application path: C:\Program Files\KeePassXC\KeePassXC.exe
Faulting module path: C:\WINDOWS\SYSTEM32\MSVCP140.dll
Report Id: 8d544add-cd4b-4d40-93d6-1bb191482994
Faulting package full name: 
Faulting package-relative application ID: 

I has been running OK until I rebooted the machine.

**Comment by droidmonkey** [comment]
Update your msvc redistributable

**Comment by jmssil** [comment]
Yes, that seems to fix it, thanks.

**Comment by drHyperion451** [comment]
To someone who had the same problem, the link to download MSVC is: https://learn.microsoft.com/es-es/cpp/windows/latest-supported-vc-redist?view=msvc-170

**Comment by ribas89** [comment]
> Github search must be broken today....
> 
> [#10925 (comment)](https://github.com/keepassxreboot/keepassxc/issues/10925#issuecomment-2179741873)

We are coming from google that's why we didn't notice your comment sorry.

**Comment by WiliTest** [comment]
Updating msvc redistributable worked. 

In my case clicking on the shortcut after installing did nothing (no error), which is confusing. 

It would be great to have an alert saying ≈ "if not working try updating msvc redistributable"

**Comment by jmssil** [comment]
Today this happened again to me in the PC where I use KeePassXC. I updated Microsoft Visual C++ Redistributable and again it fixed the issue.

I tried running KPXC in the command line but there was not output. I tried --debug-info but still there was no output. I suggest adding an initial check with a message saying that the version of the Microsoft Visual C++ Redistributable is not compatible and to update.
